### DISCUSSION TITLE
I trying to implement CI shield to my project, but it have very small documentation.

### DISCUSSION THREAD

**Discussion by nguyennamit** [codeigniter4/shield#769] [Q&A]
I trying to implement CI shield to my project, but it have very small documentation. 

My project have a website with login for clients and separate login for admin to his panel with different urls.
I wanna to use shield at this two interfaces, but i don't understand how i can load special config for each, for example: 
For /admin – use /modules/admin/config/auth.php
For /userlogin use /app/config/auth.php (default config)

These two interfaces will have a completely different view, but they can use the same users database just as different groups (for example, groups for accessing the admin panel - admin / manager, and groups for accessing the user login - client)

**Comment by datamweb** [comment]
> but it have very small documentation.

You are implementing a custom controller, we cannot explain in the documentation everything that a developer should implement.


> For /userlogin use /app/config/auth.php (default config)

You can use default items for  login clients.


> For /admin – use /modules/admin/config/auth.php

Shield doesn't have an external login page by default (users/admin), and I don't think there's a need for it. However, you can code it yourself if needed.

For the admin, according to your insistence on the separation of the interface, **you can write a custom controller.**

All things are in the documents (including, [how to add a user group](https://codeigniter4.github.io/shield/authorization/#addgroup), [how to redirect a specific group to a specific route](https://codeigniter4.github.io/shield/customization/#customize-login-redirect), [how to protect the route of administrators](https://codeigniter4.github.io/shield/install/#protect-all-pages), [how to define a new route](https://www.codeigniter.com/user_guide/incoming/routing.html), ...)
Please read the small documentation carefully, it's definitely not complicated to implement.
